### ISSUE TITLE
Smoker movement penalty can stack multiple times

### ISSUE THREAD

**Issue by alexiscoutinho** [Tsuey/L4D2-Community-Update#376]
### Description

If two or more Smokers catch a survivor in quick succession, the player can get stuck/frozen for 5+ seconds (I assume the effect is stacked for 3+ Smokers, but I only tested in Hard Eight). This is similar in effect to #146.

**Comment by Derpduck** [comment]
Not related to #146
`tongue_release_fatigue_penalty` determines the amount of slowdown applied, I assume it just adds the fatigue value onto the player each time a smoker is cleared. Maybe capping the fatigue the player can have to the cvar value would fix the issue?

**Comment by alexiscoutinho** [comment]
Yeah, they are technically unrelated. What I meant was that in both cases the survivor is helpless/vulnerable for a long time.

**Comment by Nesciuse** [comment]
the fatigue is being added to cslocaldata.fl_Stamina

https://github.com/Tsuey/L4D2-Community-Update/assets/36797145/cc4e1f15-e613-4096-a283-207c76d1cf2c
